common/src/main/java/haveno/common/crypto/KeyStorage.java

```diff
@@ -255,17 +259,20 @@ private void saveKey(SecretKey key, String alias, String fileName, String oldPas
             KeyStore keyStore = KeyStore.getInstance("PKCS12");
 
             // load from existing file or initialize new
-            try {
-                keyStore.load(new FileInputStream(path), oldPasswordChars);
-            } catch (Exception e) {
+            try (FileInputStream fileInputStream = new FileInputStream(path)) {
```

Comment:
This will cause this method to succeed even if the key store fails to load and save, in which case the caller would have no knowledge of the failure?